products/BellHybrid/BellHybridMain.cpp

```diff
@@ -68,6 +71,7 @@ int main()
 
     std::vector<std::unique_ptr<sys::BaseServiceCreator>> systemServices;
     systemServices.emplace_back(sys::CreatorFor<EventManager>([]() { return dumpLogs(); }));
+    systemServices.emplace_back(sys::CreatorFor<service::ServiceFileIndexer>());
```

Comment:
It is a perfect place to inject product-specific parameters into the file indexer.